### PR TITLE
Url refactor.

### PR DESCRIPTION
**Pull request by WlodekLipski**
Index page, table and plot pages added. Small pages refactor done.
Need to connect value from dropdown list with data on table page and
plot.

### CODE REVIEW

File: measurements/urls.py [hunk 1]
```diff
@@ -7,5 +7,6 @@
 
 urlpatterns = [
     path('', include(router.urls)),
-    path('table/', views.table)
+    path('view/', views.view),
+    path('data/', views.table)
```
Comment (mikitachab) [inline measurements/urls.py]: why not just:
```python
path('table/', views.table)
```

File: measurements/views.py [hunk 2]
```diff
@@ -1,26 +1,50 @@
...
     queryset = paginator.get_page(page)
+    context = {
+            "data_list": queryset,
+            "index" : "/",
+            "title": "Measured data",
+            "devices": devices
+        }
+    return render(request, "data_table.html", context)
+
+def view(request):
```
Comment (mikitachab) [inline measurements/views.py]: please consider more descriptive name

File: prepare_venv.sh [hunk 3]
```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
-python3 -m virtualenv venv
-source venv/bin/activate
+python3 -m venv virtualenv
```
Comment (mikitachab) [inline prepare_venv.sh]: I don't understand why you did that. `venv` and `virtualenv` a different packages. they are doing mostly the same, `venv` is prebuilt, but `virtualenv` more preferable to use and `venv` is more preferable name for virtual environment. if you had troubles with running that script please try:
```bash
pip install virtualenv
```
and please change it as it was before.

**Review by mikitachab — CHANGES_REQUESTED** [review]

**Comment by WlodekLipski** [comment]
Commit changes.

**Review by mikitachab — APPROVED** [review]

**Pull request merged**